### PR TITLE
VAR-135 | Tooltip/Popover/Dropdown for manage reservation view

### PR DESCRIPTION
**Pull request by midori0507**

### CODE REVIEW

File: src/common/popover/PopoverOverlay.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Popover from 'react-bootstrap/lib/Popover';
+import OverlayTrigger from 'react-bootstrap/lib/OverlayTrigger';
+import PropTypes from 'prop-types';
+
+const PopoverOverlay = ({
+  content, title, trigger, children, placement
+}) => (
+  <div className="app-PopoverOverlay">
+
```
Comment (rBoost) [inline src/common/popover/PopoverOverlay.js]: Why does this component have so many empty lines in it?

File: src/domain/reservation/manage/comment/ManageReservationsComment.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import PopoverOverlay from '../../../../common/popover/PopoverOverlay';
+import commentIcon from '../../../../../app/assets/icons/comment.svg';
+
+const ManageReservationsComment = ({ comments }) => (
+  <div className="app-ManageReservationComments">
+
```
Comment (rBoost) [inline src/domain/reservation/manage/comment/ManageReservationsComment.js]: Same here, why so many empty lines?

File: app/i18n/messages/sv.json [hunk 3]
```diff
@@ -308,7 +308,13 @@
   "ManageReservationsList.dateAndTimeHeader": "När",
   "ManageReservationsList.statusHeader": "Status",
   "ManageReservationsList.pinHeader": "PIN",
+
   "ManageReservationsList.actionsHeader": "Välj åtgärd",
+  "ManageReservationsList.actionLabel.information":"Information",
+  "ManageReservationsList.actionLabel.approve":"Approve",
```
Comment (jimppa) [inline app/i18n/messages/sv.json]: Godkänn

File: app/i18n/messages/sv.json [hunk 4]
```diff
@@ -308,7 +308,13 @@
...
+  "ManageReservationsList.actionLabel.deny":"Deny",
```
Comment (jimppa) [inline app/i18n/messages/sv.json]: Förneka

File: app/i18n/messages/sv.json [hunk 5]
```diff
@@ -308,7 +308,13 @@
...
+  "ManageReservationsList.actionLabel.edit":"Edit",
```
Comment (jimppa) [inline app/i18n/messages/sv.json]: Redigera

File: app/i18n/messages/fi.json [hunk 6]
```diff
@@ -316,7 +316,13 @@
   "ManageReservationsList.dateAndTimeHeader": "Milloin",
   "ManageReservationsList.statusHeader": "Tila",
   "ManageReservationsList.pinHeader": "PIN",
+  
   "ManageReservationsList.actionsHeader": "Valitse toiminta",
+  "ManageReservationsList.actionLabel.information":"Information",
+  "ManageReservationsList.actionLabel.approve":"Approve",
```
Comment (jimppa) [inline app/i18n/messages/fi.json]: Hyväksy

File: src/common/popover/PopoverOverlay.js (hunk 1 above)
Comment (midori0507) [reply src/common/popover/PopoverOverlay.js]: Fixed

File: app/i18n/messages/fi.json [hunk 7]
```diff
@@ -316,7 +316,13 @@
...
+  "ManageReservationsList.actionLabel.deny":"Deny",
```
Comment (jimppa) [inline app/i18n/messages/fi.json]: Hylkää

File: app/i18n/messages/fi.json [hunk 8]
```diff
@@ -316,7 +316,13 @@
...
+  "ManageReservationsList.actionLabel.edit":"Edit",
```
Comment (jimppa) [inline app/i18n/messages/fi.json]: Muokkaa

**Pull request merged**